htdocs/reception/card.php

```diff
@@ -1263,8 +1263,7 @@
 						$stock = +$product->stock_warehouse[$dispatchLines[$indiceAsked]['ent']]->real; // Convert to number
 					}
 					$deliverableQty = $dispatchLines[$indiceAsked]['qty'];
-					$cost_price = $dispatchLines[$indiceAsked]['pu'];
-
+					$cost_price = price($dispatchLines[$indiceAsked]['pu']);
```

Comment:
A php variable must be in php format (example 1.2), the function price is used to format in the user language format (example 1,2 in france).
So the call to price must be done only for the output on screen and not to assign variable.
Can you move it later when we do the print $cost_price ?